README.md

```diff
@@ -27,37 +27,40 @@ See the [list of releases](../../releases).
-	* Recommended extensions: [JSON](https://secure.php.net/json), [GMP](https://secure.php.net/gmp) (for API access on platforms < 64 bits), [IDN](https://secure.php.net/intl.idn) (for Internationalized Domain Names), [mbstring](https://secure.php.net/mbstring) (for Unicode strings), [iconv](https://secure.php.net/iconv) (for charset conversion), [ZIP](https://secure.php.net/zip) (for import/export), [zlib](https://secure.php.net/zlib) (for compressed feeds)
+	* Recommended extensions: [JSON](https://secure.php.net/json), [GMP](https://secure.php.net/gmp) (for API access on non 64 bit platforms), [IDN](https://secure.php.net/intl.idn) (for Internationalized Domain Names), [mbstring](https://secure.php.net/mbstring) (for Unicode strings), [iconv](https://secure.php.net/iconv) (for charset conversion), [ZIP](https://secure.php.net/zip) (for import/export), [zlib](https://secure.php.net/zlib) (for compressed feeds)
 * MySQL 5.5.3+ (recommended), or SQLite 3.7.4+, or PostgreSQL 9.2+
 * A recent browser like Firefox / IceCat, Internet Explorer 11 / Edge, Chromium / Chrome, Opera, Safari.
 	* Works on mobile
 
 ![FreshRSS screenshot](docs/img/FreshRSS-screenshot.png)
 
-# Documentation
-* https://freshrss.github.io/FreshRSS/en/
+# [Documentation](https://freshrss.github.io/FreshRSS/en/)
+* [User documentation](https://freshrss.github.io/FreshRSS/en/users/02_First_steps.html) so you can discover all the power of FreshRSS
+* [Administrator documentation](https://freshrss.github.io/FreshRSS/en/admins/01_Index.html) for detailed installation and maintenance related tasks
+* [Developer documentation](https://freshrss.github.io/FreshRSS/en/developers/01_First_steps.html) to guide you in the source code of FreshRSS and to help you if you want to contribute
+* [Contributor guidelines](https://freshrss.github.io/FreshRSS/en/contributing.md) for those who want to help improve FreshRSS
 
 # [Installation](https://freshrss.github.io/FreshRSS/en/admins/02_Installation.html)
 1. Get FreshRSS with git or [by downloading the archive](https://github.com/FreshRSS/FreshRSS/archive/master.zip)
-2. Dump the application on your server (expose only the `./p/` folder)
-3. Add write access on `./data/` folder to the webserver user
+2. Put the application somewhere on your server (expose only the `./p/` folder to the web)
+3. Add write access on the `./data/` folder for the webserver user
 4. Access FreshRSS with your browser and follow the installation process
 	* or use the [Command-Line Interface](cli/README.md)
-5. Everything should be working :) If you encounter any problem, feel free [contact us](https://github.com/FreshRSS/FreshRSS/issues).
+5. Everything should be working :) If you encounter any problem, feel free to [contact us](https://github.com/FreshRSS/FreshRSS/issues).
```

Comment:
```suggestion
5. Everything should be working :) If you encounter any problems, feel free to [contact us](https://github.com/FreshRSS/FreshRSS/issues).
```

(I think it sounds better as `if you encounter a problem` or `if you encounter any problems`.)